### DISCUSSION TITLE
Fundamental question

### DISCUSSION THREAD

**Discussion by rouighirafik** [EA31337/EA31337-strategies#170] [General]
Hey there,
Just found your gem so thanks a lot for the huge work, very impressive and kudos.
I have a very basic and fundamental question: Do the trades open based on the combination of the indicators signals or each single timeframe will follow a single indicator signal?
If No, how feasible is it to make a combination?

**Comment by kenorb** [comment]
Each instance of strategy is assigned to specific timeframe, then EA from the higher perspective can manage multiple strategies at once.
However assignment of strategy to timeframe is not set in stone, as it can still add and read data from multiple indicators which can work on different timeframes, in order to generate strategy's signal (e.g. RSI on M15 to signal, and RSI on H1 to filter out the signal).
In general, strategy can return open trade signal based on its indicators running on specific timeframes, but this can be flexible.
If it's still not clear, if you could clarify the question further more.
